src/shapes/text.class.js

```diff
@@ -1203,12 +1203,16 @@
      */
     _getFontDeclaration: function(styleObject, forMeasuring) {
       var style = styleObject || this;
+      var fontFamily = style.fontFamily === undefined ||
+      style.fontFamily.includes('\'') ||
+      style.fontFamily.includes('"')
```

Comment:
sorry to be late with this comment.
includes is something forever supported by IE, just recently included in firefox and chrome as per new specs.
I think there is a string method that is supported by everyone and should be es5 compatible.
FabricJS is es5 and .includes is not.